src/index.ts

```diff
@@ -10,7 +11,14 @@ const server = new ApolloServer({
 
 const port = 3001
 
-server.listen({ port }).then(({ url }) => {
-    // eslint-disable-next-line no-console
-    console.log(`🚀  Server ready at ${url}`)
-})
+async function getUrl() {
```

Comment:
As a general rule, "getters" should not have side effects. Here, we have a very big side effect: starting a web server, and then we're not even using the `url` we got.

At minimum I suggest a rename

```suggestion
async function startServer() {
```

Reply:
@ann-kilzer  Could you take another look at it? I refactored it with your suggestions.